ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/EthGetProofTest.java

```diff
@@ -176,7 +182,7 @@ private GetProofResult generateWorldState() {
     final Wei balance = Wei.of(1);
     final Hash codeHash =
         Hash.fromHexString("0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470");
-    final long nonce = 1;
+    final long nonce = Long.parseUnsignedLong("18446744073709551614");
```

Comment:
is it worth spelling it out with a const name eg MAX_UNSIGNED_LONG_VALUE

Reply:
Thank you! I've changed them now and checked again, I think now all occurences for the nonce have been changed

Reply:
I've change it now to use constants in the tests